exe/Server/Server.hs

```diff
@@ -61,6 +66,16 @@ data Chain = Chain
 -- matter for a test server.
 newtype Chains = Chains { getChains :: MVar (Map Text Chain) }
 
+type ServerApi
+    = "chains" :> Capture "chain" Text :> ChainSubmitEndpoint
+ :<|> "chains" :> Capture "chain" Text :> ChainSinceEndpoint
```

Comment:
I'd abstract the common path, since if they're *not* common everything breaks. I.e.:

```
"chains" :> Capture "chain" Text :> (ChainSubmitEndpoint :<|> ChainSinceEndpoint)
```